packages/frontend/src/components/page-not-found/PageNotFound.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Translate } from "react-localize-redux";
+import styled from "styled-components";
+
+import textBackgroundImage  from "../../images/bg-landing-patterned.svg";
+import FormButton from "../common/FormButton";
+import Container from "../common/styled/Container.css";
+
+
+
+const StyledContainer = styled.div`
+  &&& {
+    margin: 35px 5px 0 5px;
+    position: relative;
+    text-align: center;
+
+    @media (max-width: 767px) {
+      margin: 0;
+      padding: 0 20px;
+      overflow: hidden;
+      margin-top: -13px;
+    }
+
+    .small-centered {
+      padding-top: 20px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
```

Comment:
We can move this to the top `StyledContainer`:

`&&& {
    text-align: center;
    padding-top: 20px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}`